tests/mgbench/workloads/base.py

```diff
@@ -200,12 +201,15 @@ def _set_local_index_file(self) -> None:
 
     def _set_url_index_file(self) -> None:
         if self.URL_INDEX_FILE is not None:
-            self._url_index = self.URL_INDEX_FILE.get(self._vendor, None)
+            for vendor, url in self.URL_INDEX_FILE.items():
+                if vendor in self._vendor:
+                    self._url_index = url
+                    return
         else:
             self._url_index = None
```

Comment:
```suggestion
        else:
            self._url_index = None
```
`self._url_index = None`